app/app.js

```diff
@@ -0,0 +1,61 @@
+import $ from "jquery";
+import * as tournament from "./tournament/tournament";
+import * as player from "./tournament/player";
```

Comment:
On these 2 cases, I think it´s more natural to import {Player} and {Tournament} directly.